bigbluebutton-html5/imports/ui/components/app/component.jsx

```diff
@@ -162,7 +163,6 @@ class App extends Component {
             {this.renderMedia()}
             {this.renderActionsBar()}
           </div>
-          {this.renderSidebar()}
```

Comment:
Don't delete the SideBar box with its styles please, we might need it later.
Just create a separate function to render closed-captions similar to the renderSideBar() and pass closed-captions object there instead of the sidebar: https://github.com/JaeeunCho/bigbluebutton/blob/92bc296bc05fa0cc961bb793e81b4a28010cfa80/bigbluebutton-html5/imports/ui/components/app/container.jsx#L76
The function can be something like this:
```
   renderClosedCaptions() {
     const { captions } = this.props;

    if (captions) {
      return (
        <div className={styles.closedCaptions}>
          {captions}
        </div >
      );
     }
   }
```